src/app/(proper_react)/(redesign)/(public)/breaches/page.tsx

```diff
@@ -30,6 +30,9 @@ export function generateMetadata() {
 
 export default async function Page() {
   const allBreaches = await getBreaches();
+  const breachesNewestFirst = allBreaches.sort((a, b) => {
```

Comment:
Nit, but `.sort` modifies the array in place, so by re-assigning it to a new variable, people might be mislead that the original (`allBreaches`) is unmodified when it's not. `.toSorted` is the version that does not modify the original.

(It was only added recently, but since this runs on the server where we're running Node 20, that should be fine. Alternatively, you could just call `.sort` and not re-assign to `breachesNewestFirst`, but just continue working with `allBreaches`.)

Reply:
Updated here:
[135736436744f0e19afac9a7b4b770ee29f084fc](https://github.com/mozilla/blurts-server/pull/4757/commits/135736436744f0e19afac9a7b4b770ee29f084fc)